source/content/modules-plugins-known-issues.md

```diff
@@ -507,6 +507,15 @@ The solutions [outlined in the EWWW documentation](https://docs.ewww.io/article/
 
 <hr />
 
+### [InfiniteWP](https://infinitewp.com)
+**Issue 1**: Installing the InfiniteWP admin panel in a Pantheon hosted site is not possible because they are hardcoding the DB creds and using custom port in the url due to platform limitations.
```

Comment:
@carl-alberto this makes it sound like the plugin don't won't work on our platform, yet there's a **Solution** on like 515. Is the "admin panel" not required to use the plugin? Please clarify this phrasing.

Reply:
@alexfornuto Not sure how to word it correctly but there are 2 parts of this plugin. Admin part that cannot be installed on the platform because it needs a custom port and requires the db creds to be hardcoded so it is best to host the admin elsewhere or use the author's Premium service

Reply:
@carl-alberto would an admin panel running in a site on the user's local computer work?